### PR TITLE
Added personal linkfree

### PR DESCRIPTION
**Pull request by hasnainmakada-99**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1319"></a>

### CODE REVIEW

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
There is no icon named `portfolio`. Change it to `globe`.

**Comment by hasnainmakada-99** [comment]
Ok

File: public/data/hasnainmakada-99.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/hasnain-makada-5b47271aa/",
+      "icon": "linkedIn"
+    },
+    {
+      "name": "Check out my portfolio",
+      "url": "https://hasnainmakada-99.github.io",
+      "icon": "portfolio"
```
[the remark this answers is not in the document]
Comment (hasnainmakada-99) [reply public/data/hasnainmakada-99.json]: The changes have been made

**Review by Panquesito7 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
LGTM 👍  Thanks for your contribution! 🚀

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**